CHANNELS.md

```diff
@@ -60,9 +60,16 @@ Channel | Type | Purpose | Role Permissions
 Channel | Type | Purpose | Role Permissions
 --- | --- | --- | ---
 #elite-2018 | :speech_balloon: | Discussion of CyberStart Elite 2018, particularly to keep spoiler-y discussions separate | @everyone denied, @Veteran 2018 can read/write
-#cloud-ctf | :speech_balloon: | Discussion of the CyberStart Cloud CTF Beta | @everyone denied, @Cloud CTF can read/write
-Cloud CTF | :microphone: | Corresponding voice channel for #cloud-ctf | @everyone denied, @Cloud CTF can talk
 CyberStart Elite | :microphone: | Corresponding voice channel for #elite-2018 | @everyone denied, @Veteran 2018 can join and talk
```

Comment:
This is now out of date, and should probably be changed in this commit. (I'd also take the opportunity to move the VC down next to `Elite 2019` to match position in the server) Otherwise this can go to voting soon.
```suggestion
Elite 2018| :microphone: | Corresponding voice channel for #elite-2018 | @everyone denied, @Veteran 2018 can join and talk
```